src/agent/v8/inspector-debugapi.ts

```diff
@@ -108,36 +83,28 @@ export class InspectorDebugApi implements debugapi.DebugApi {
     this.config = config;
     this.fileStats = jsFiles;
     this.sourcemapper = sourcemapper;
-    this.scriptMapper = {};
-    this.v8 = this.createV8Data();
-  }
-
-  /** Creates a new V8 Debugging session and the related data. */
-  private createV8Data(): V8Data {
-    const session = new inspector.Session();
-    session.connect();
-    session.on('Debugger.scriptParsed', script => {
-      this.scriptMapper[script.params.scriptId] = script.params;
-    });
-    session.post('Debugger.enable');
-    session.post('Debugger.setBreakpointsActive', {active: true});
-    session.on('Debugger.paused', message => {
-      try {
-        this.handleDebugPausedEvent(message.params);
-      } catch (error) {
-        this.logger.error(error);
+    this.inspector = new V8Inspector(
```

Comment:
useWellFormattedUrl is in here twice.  One of them is mislabeled.

Reply:
Done.